index.bs

```diff
@@ -601,6 +602,14 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     By definition, the [=credential private key=] is always exclusively controlled by the [=authenticator=]. In the case of a
     [=resident credential=], the [=authenticator=] might offload storage of wrapped key material to the
     [=client device=], but the [=client device=] is not expected to offload the key storage to remote entities (e.g., [=[WRP]=] Server).
+    An authenticator is said to be using [=client-side credential storage modality=] when generating and storing a [=Resident Credential=].
+
+: <dfn>Non-Resident Credential</dfn>
+: <dfn>Server-side-resident Public Key Credential Source</dfn>
```

Comment:
The terminology section is sorted, so I think we should move this to between "RP ID" and "Test of User Presence". It makes a lot of sense to have this directly following the Client-side-resident Public Key Credential Source definition, but I don't think it's worth breaking the ordering.

Reply:
Ok - I did noticed the ordering was alphabetical, and had chosen to put them adjacent. No problem with moving it though. Done.